src/components/StatueListContainer.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from "react";
+import StatueList from "./Statuelist";
+import SlaverStatues from "../Statues/slavers";
+
+export class StatueListContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      statues: SlaverStatues,
+    };
+  }
+
+  render() {
+    return (
+      <div>
+        <h1>Statues</h1>
+        <StatueList statue={this.state.statues} />
+      </div>
+    );
+  }
+}
+
+export default StatueListContainer;
```

Comment:
How else could this (and the other) exports be done?

Reply:
We tried to do a named export but it did not seem to work ... 
We will look into it again a bit later

Reply:
We tried to do a named export but it did not seem to work ... 
We will look into it again a bit later